src/WingetCreateCore/Models/InstallerManifestModels.cs

```diff
@@ -159,8 +160,22 @@ public partial class InstallerSwitches
             get { return _additionalProperties; }
             set { _additionalProperties = value; }
         }
-    
-    
+
+        /// <summary>Gives the criteria for determining whether two instances of InstallerSwitches objects are equal</summary>
+        public override bool Equals(object obj)
+        {
+            if (obj == null || GetType() != obj.GetType())
+                return false;
+
+            InstallerSwitches other = (InstallerSwitches)obj;
+            return Silent == other.Silent &&
+                SilentWithProgress == other.SilentWithProgress &&
+                Interactive == other.Interactive &&
+                InstallLocation == other.InstallLocation &&
+                Log == other.Log &&
+                Upgrade == other.Upgrade &&
+                Custom == other.Custom;
+        }

```

Comment:
The InstallerManifestModel.cs file cannot be modified as it is autogenerated based on the latest schema. 

You will need to move these functions to `WingetCreateCore/Models/InstallerPartials` and follow the pattern there. Make sure you do that for the other `Equals` methods so that this file has no changes and remove the `using System.Linq` statement.

Reply:
Shifted the methods to `WingetCreateCore/Models/InstallerPartials` and reset changes to `InstallerManifestModel.cs`